### DISCUSSION TITLE
Resume Bundle x ATS Scoring

### DISCUSSION THREAD

**Discussion by abhirup0199** [recodehive/resume-pitch#92] [Q&A]
Please check both of my resumes - 
[Abhirup_CV.pdf](https://github.com/user-attachments/files/17006136/Abhirup_CV.pdf)
[Abhirup_CV_main.pdf](https://github.com/user-attachments/files/17006137/Abhirup_CV_main.pdf)

**Comment by sanjay-kv** [comment]
[abhi.pdf](https://github.com/user-attachments/files/17089285/abhi.pdf)
[ATS Version of CV (23).docx](https://github.com/user-attachments/files/17089286/ATS.Version.of.CV.23.docx)

@abhirup0199
